tests/x11regressions/evolution/evolution_timezone_setup.pm

```diff
@@ -38,8 +39,15 @@ sub run {
     send_key "alt-s";
     wait_still_screen 3;
     send_key "ret";
-    send_key_until_needlematch("timezone-asia-shanghai", "up")
-      || send_key_until_needlematch("timezone-asia-shanghai", "down");
+    if (sle_version_at_least('12-SP3')) {
```

Comment:
hmm. I would prefer if you did this without hardcoding versions - but by means of a check_screen. Because leap will get this update too I assume - and then you end up with strange version checks.